source/includes/fact-snmp-configuration-files.rst

```diff
@@ -1,18 +1,50 @@
-MongoDB Enterprise contains the following configuration files to
-support SNMP:
+MongoDB Enterprise includes the following SNMP configuration files:
 
 - ``MONGOD-MIB.txt``:
 
   The management information base (MIB) file that defines MongoDB's
-  SNMP output.
+  SNMP output. On Ubuntu and Debian platforms, this file is shipped
+  compressed as ``MONGOD-MIB.txt.gz`` and must be decompressed with
+  ``gunzip``.
 
 - ``mongod.conf.subagent``:
 
   The configuration file to run |mongod-program| as the SNMP
   subagent. This file sets SNMP run-time configuration options,
-  including the ``AgentX`` socket to connect to the SNMP master.
+  including the ``agentXSocket`` to connect to the SNMP master.
 
 - ``mongod.conf.master``:
 
   The configuration file to run |mongod-program| as the SNMP
-  master. This file sets SNMP run-time configuration options.
+  master. This file sets SNMP run-time configuration options,
+  including the ``agentaddress`` to run on.
+
+The MongoDB SNMP configuration files are provided with your MongoDB
+Enterprise installation, as follows:
+
+- If you installed MongoDB Enterprise via a package manager, these files
```

Comment:
Also included in the SNMP MongoDB Windows page

https://docs-mongodbcom-staging.corp.mongodb.com/andrew.feierabend/DOCS-5839-clarify-snmp-trap-and-bind/tutorial/monitor-with-snmp-on-windows.html#configuration-files

Reply:
You're right. Fixed by moving the Windows-specific instructions to their own file. Will revisit in a separate ticket.